### PR TITLE
CRUD resources admin

### PR DESCRIPTION
**Pull request by TrinityChristiana**
**QA**

- [x] After pulling this down go to  `http://127.0.0.1:8000/admin/resources/`
- [x] You should be taken to a page with a list of resources
- [x] The header should read "All Resources"
- [x] Click the Create button
- [x] You should be taken to. a page with an empty form
- [x] The page should also have a header that reads "Creating Resource"
- [x] There should be a submit button that reads "Create"
- [x] Create a resource
- [x] After creation, you should be taken to the list view with an alert that says "Resource Created"
- [x] Check to make sure you can search for your new resource in the List view
- [x] find a resource in the resource list and click the name
- [x] It should take you to a page that has a form populated with the data from that resource
- [x] There should be a submit button that reads "Update"
- [x] The page should also have a header that reads "Updating Resource"
- [x] After creation, an alert that says "Resource Created" appears
- [x] Go to the following page:`http://127.0.0.1:8000/admin/resources/`
- [x] Find the resource you updated and it should have changed
- [x] find a resource in the resource list and click the name
- [x] It should take you to a page that has a form populated with the data from that resource
- [x] There should be a delete button that reads "Delete"
- [x] Click that button
- [x] After deletion, you should be taken to the list view with an alert that says "Resource Deleted"
- [x] Try to find the item you deleted. It should be gone.

### CODE REVIEW

File: backend/h4tcsacadmin/admin/__init__.py [hunk 1]
```diff
@@ -34,3 +37,5 @@ def dashboard_view(self, request):
 admin_site.register(ReportType)
 admin_site.register(Resource)
 admin_site.register(Sponser)
+admin_site.register(ListBar)
```
Comment (TrinityChristiana) [inline backend/h4tcsacadmin/admin/__init__.py:40]: Re-added this so we can easily add data if needed

File: backend/h4tcsacapi/settings.py [hunk 2]
```diff
@@ -44,9 +44,8 @@
     env_map["API_HOST"],
     "localhost"
 ]
-CORS_ORIGIN_WHITELIST = [
-    env_map["FRONTEND_ORIGIN"],
-]
+
+CORS_ALLOW_ALL_ORIGINS = True
```
Comment (TrinityChristiana) [inline backend/h4tcsacapi/settings.py:48]: Added this so that there are no more cors errors

File: backend/h4tcsacapi/settings.py [hunk 3]
```diff
@@ -59,9 +58,9 @@
     'django.contrib.staticfiles',
     'rest_framework',
     'h4tcsacapp',
+    'h4tcsacadmin',
     'corsheaders',
-    'h4tcsacrecertify'
-    'h4tcsacadmin'
+    'django_extensions'
```
Comment (TrinityChristiana) [inline backend/h4tcsacapi/settings.py:63]: This is an app that allows you to check a lot of Django configs that you can't directly see.

File: backend/h4tcsacapp/migrations/0003_alter_contact_phone_number.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.4 on 2022-04-28 20:10
+
+from django.db import migrations, models
```
Comment (TrinityChristiana) [inline backend/h4tcsacapp/migrations/0003_alter_contact_phone_number.py:3]: Had to update the Phone number to big int to hole larger numbers for db

File: backend/h4tcsacadmin/views/resources.py [hunk 5]
```diff
@@ -0,0 +1,100 @@
+from django.http import HttpResponseRedirect
+from django.shortcuts import get_object_or_404, render
+from django.urls import path, reverse
+
+
+class CustomDjangoViews():
```
Comment (TrinityChristiana) [inline backend/h4tcsacadmin/views/resources.py:6]: THis can be used to quickly create admin crud on tables

**Review by EugeneTerry — APPROVED** [review]
works

**Pull request merged**